src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -287,9 +298,12 @@ private void StartLogging()
 
         private string GetLogDirPath()
         {
-            string logDir = !string.IsNullOrEmpty(LogPath)
-                ? Path.GetDirectoryName(LogPath)
-                : Path.GetDirectoryName(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
+            string logDir = Path.GetDirectoryName(LogPath);
+            if (string.IsNullOrEmpty(logDir))
+            {
+                logDir = Directory.GetCurrentDirectory();
+                // Path.GetDirectoryName(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
```

Comment:
I really don't know why this was the previous safe default, do you Patrick?

Reply:
While neither are particularly safe, I'd say the location of the assembly is marginally safer of the two. Process working directory is often somewhat arbitrary, could be a project directory, could be system32

Maybe `Path.GetTempPath()` + `\PowerShellEditorServices` would be good?

Reply:
I vaguely remember having an issue with using the temp path, not sure why...